cip-constitution-tech/README.md

```diff
@@ -0,0 +1,255 @@
+This is essential for the governance model
+
+Like the Cardano constitution this standard impacts every ada holder.
+
+### Safety
+
+Without describing best practices for the form and handling of the constitution, we risk the constitution document being stored in an insecure manner.
+By storing the constitution on a decentralized platform, we can ensure it's immutability and permissionless access.
+This is a step to ensuing the longevity and accessibility of each constitution iteration.
+
+### Interoperability
+
+By defining a file extension and formatting rules for the constitution we ensure that tooling reading and writing the constitution will be interoperable.
+Furthermore we aim to make roles constitution iteration comparison tools easy, by minimizing formatting and style changes between iterations.
+This will reduce compatibility issues between tools, promoting the accessibility of the constitution.
+
+### Usability
+
+Rich text formatting greatly enhances the readability of text, especially in large complex documents.
+Without the ability to format text, it could easily become cumbersome to read, negatively effecting the accessability of the Cardano constitution.
+
+## Specification
+
+The following specification should be applied to all constitution iterations.
+This standard could be augmented in the future via a separate CIP which aims to replace this one.
+
+### File Type
+// todo - any suggestions?
+
+### Hashing
+
+When supplying a constitution hash digest to chain, the algorithm used MUST be Blake2b-256.
+Before creating a hash digest the constitution plain text must be in its raw un-rendered format.
+
+- add a way to change algorithm?
+
+### Storage
+
+The each ratified constitution MUST be stored, immutably on a distributed storage mechanism (such as IPFS).
+Where backups can be easily made in a permission less manner by interested parties.
+Must be easily accessible, normal tooling can support it.
+Authors SHOULD NOT specify the use of centralized gateways to access the constitution plain text.
+
+### Rich Text Formatting
+
+The constitution text MAY include a subset of markdown text styling.
+Tooling rendering the constitution for users, should recognize these and render them faithfully.
+
+#### Headers
+
+```md
+# H1
+## H2
+### H3
+#### H4
+##### H5
+###### H6
+
+Alternatively, for H1 and H2, an underline-ish style:
+
+Alt-H1
+======
+
+Alt-H2
+------
+```
+
+#### Emphasis
+
+```md
+Emphasis, aka italics, with *asterisks* or _underscores_.
+
+Strong emphasis, aka bold, with **asterisks** or __underscores__.
+
+Combined emphasis with **asterisks and _underscores_**.
+
+Strikethrough uses two tildes. ~~Scratch this.~~
+```
+
+#### Lists
+
+```md
+1. First ordered list item
+2. Another item
+⋅⋅* Unordered sub-list. 
+1. Actual numbers don't matter, just that it's a number
+⋅⋅1. Ordered sub-list
+4. And another item.
+```
+
+#### Links
+
+```md
+[I'm an inline-style link](https://www.google.com)
```

Comment:
Is it standard that an in-line link will display the hyperlink when you hover?  Although unlikely, it feels like a security flaw if a malicious link gets hidden in the rendered version of the adopted constitution.

Reply:
[It depends on the tool rendering the link](https://github.com/Ryun1/CIPs/pull/9/files)

I think we can assume a constitution with malicious links shouldnt get voted in